ui/src/jobs/components/__tests__/MoreInfo.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { fromJS } from 'immutable';
+import MoreInfo from '../MoreInfo';
+
+describe('MoreInfo', () => {
+  it('renders with urls', () => {
+    const urls = fromJS([
+      {
+        value: 'url1',
+      },
+      {
+        value: 'url2',
+      },
+    ]);
+    const wrapper = shallow(<MoreInfo urls={urls} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```

Comment:
what if 

```js
<MoreInfo />
```

Reply:
https://github.com/inspirehep/inspirehep/pull/212/files#diff-545b14684e0ea3d99a7df1f6b0ecd2d2R24

Reply:
As in add a test with <MoreInfo/> ? The snapshot would be empty for this, does it make sense?

Reply:
if urls is empty, doesn't the https://github.com/inspirehep/inspirehep/pull/212/files#diff-545b14684e0ea3d99a7df1f6b0ecd2d2R21 complain?